Examples/Swift/ViewController.swift

```diff
@@ -165,34 +165,36 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         guard let route = self.currentRoute else { return }
 
         exampleMode = .styled
-
-        let navigationViewController = NavigationViewController(for: route, locationManager: locationManager())
-        navigationViewController.navigationDelegate = self
         
-        // Set a custom style URL
-        navigationViewController.mapView?.styleURL = URL(string: "mapbox://styles/mapbox/navigation-guidance-day-v1")
-
-        let style = Style()
+        let style = DefaultStyle()
```

Comment:
It seems odd to be creating a style from scratch each time we need to refer to it. If `Style.default` was too much, how about making the singleton a global?